tests/test_http_client.rs

```diff
@@ -0,0 +1,66 @@
+#[cfg(all(any(unix, windows), feature = "http-client"))]
+#[cfg(test)]
+mod test_http_client {
+    use anyhow::Error;
+    use fluvio_future::http_client::{self, ResponseExt, StatusCode};
+    use fluvio_future::test_async;
+
+    static SERVER: &str = "https://127.0.0.1:7879";
+
+    #[test_async]
+    async fn simple_test() -> Result<(), Error> {
+        let res = http_client::get(SERVER).await;
```

Comment:
minor nit.  use [portpicker](https://crates.io/crates/portpicker) to remove hard code port?

Reply:
port is dynamic now